mesheryctl/internal/cli/root/components/search.go

```diff
@@ -67,7 +46,7 @@ mesheryctl components search [query-text]
 	RunE: func(cmd *cobra.Command, args []string) error {
 		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
 		if err != nil {
-			log.Fatalln(err, "error processing config")
+			utils.Log.Error(err)
```

Comment:
Seems like an opportunity for a centralized MeshKit Error Code

Reply:
Tested. This behavior works well given the separate handling of creating a meshconfig.